### DISCUSSION TITLE
Feature Request: GCMP=2

### DISCUSSION THREAD

**Discussion by andrerav** [gopro/labs#586] [Feature Requests]
Hi,

First of all, thanks for all the awesome work on GoPro Labs. It's simply the beez kneez and the cats meow. I love it. It's gotten me some absolutely amazing footage this summer. 

Now, I'm an avid user of GCMP=1 for MTB footage. I absolutely love how I can crank the max ISO up without worrying about my footage turning to pink mush in the evening while riding a bumpy trail. The way GCMP tries to optimize for hypersmooth performance is fantastic too. 

However -- usually I shoot in 30fps, and I've become accustomed to setting EXPX=200 (or thereabouts) to ensure good stabilization. However, after doing some nerdy experimentation, I have found a very simple and very effective way to get [very stable MTB footage](https://www.youtube.com/watch?v=fJw53aIJI08) even if the shutter speed dips down into 1/60 territory.

So long story short -- I would like to request a new GCMP mode (maybe GCMP=2 or something like that) where there is _no_ optimization of shutter speed to aid in stabilization. I.e, that with changing light conditions, the shutter speed will simply decrease until reaching EXPX before the ISO starts increasing from minimum ISO. The goal for this mode would be to completely ignore hypersmooth performance and simply minimize ISO noise as much as possible. 

I hope that this nerdy request will be considered for a future Labs release 🙏 Again, thank you so much for the increcible work with GoPro Labs!

**Comment by dnewman-gpsw** [comment]
This is a good idea, and I've wanted it too.  The camera can range in ISO and range in shutter speed, but it doesn't have prioritize lowest ISO mode (i.e. increase shutter exposure time first.)  Many controls in Labs exposes existing features that don't appear in the UI.  Setting EXPX and GCMP actually have implemented camera controls to exploit.  This particular request does not.  LOGB and WIDE didn't have existing implementations, pure Labs hackery, so it can be done.  Messing with exposure rules is just harder.  I have bumped it up the wish list.
